odoo/addons/test_impex/tests/test_load.py

```diff
@@ -1255,3 +1255,167 @@ def test_unique_pair(self):
             m.group(2).split(', '),
             ['Value2', 'Value3']
         )
+
+class test_inherits(ImporterCase):
+    """ The import process should only assign a new xid (derived from the
+    childs') if the child is being created and triggers the creation of the
+    parent
+    """
+    model_name = 'export.inherits.child'
+    def get_xid(self, record):
+        return self.env['ir.model.data'].search([
+            ('model', '=', record._name),
+            ('res_id', '=', record.id),
+        ])
+
+    def test_create_no_parent(self):
+        xids_before = self.env['ir.model.data'].search([])
+        r = self.import_(['id', 'value_parent', 'value'], [
+            ['xxx.child', '0', '1'],
+        ])
+        xids_after = self.env['ir.model.data'].search([])
```

Comment:
What is the purpose of `xids_before` and `xids_after`?

Reply:
Pretty sure I forgot to remove it after I extracted `get_xid`. That was an early way to get the list of newly created xids. Not sure why my ide didn't warn that they were unused.